### PR TITLE
Add Spanish Translations

### PR DESCRIPTION
**Pull request by gsegovia2018**

### CODE REVIEW

File: po/es_NI.po [hunk 1]
```diff
@@ -442,8 +442,10 @@ msgid ""
 msgid_plural ""
 "The earliest transaction date found in this book is %s. Based on the "
 "selection made above, this book will be split into %d books."
-msgstr[0] ""
-msgstr[1] ""
+msgstr[0] "La fecha de transacción más temprana que se encuentra en este libro es% s. Basado en la "
+"selección realizada anteriormente, este libro se dividirá en% d libro."
```
Comment (jralls) [inline po/es_NI.po]: There's a spacing error here and in the following line: It should be `en %d`, not `en% d`.

Comment (ordtrogen) [reply po/es_NI.po]: libro  --> libros (?)

File: po/es_NI.po [hunk 2]
```diff
@@ -467,6 +469,8 @@ msgid ""
 "The book will be created with the title %s when you click on 'Apply'. Click "
 "on 'Back' to adjust, or 'Cancel' to not create any book."
 msgstr ""
+"El libro será creado cuando el título %s cuando pinches en 'Aplicar'. Pincha"
```
Comment (ordtrogen) [inline po/es_NI.po]: No es "cuando el título" sino "con el título", no?

File: po/es_NI.po [hunk 3]
```diff
@@ -539,7 +545,7 @@ msgstr "No"
 #: ../gnucash/gnome-utils/dialog-options.c:690
 #: ../gnucash/gnome-utils/gnc-tree-view-account.c:903
 msgid "Placeholder"
-msgstr ""
+msgstr "Contenedor"
```
Comment (ordtrogen) [inline po/es_NI.po]: marcador de posición   ?

File: po/es_NI.po [hunk 4]
```diff
@@ -582,12 +588,12 @@ msgstr "Impuestos"
...
 #, fuzzy
 msgid "Tax Payment"
-msgstr "Pago"
+msgstr "Pago de impuestos"
 
 #: ../gnucash/gnome/assistant-loan.c:115
 #, fuzzy
 msgid "Insurance"
-msgstr "Interéses"
+msgstr "Intereses"
```
Comment (ordtrogen) [inline po/es_NI.po]: "Insurance" es "Seguro". Where did you get "Intereses"?

File: po/es_NI.po [hunk 5]
```diff
@@ -820,7 +826,7 @@ msgstr "Acciones"
 
 #: ../gnucash/gnome/assistant-stock-split.c:781
 msgid "You don't have any stock accounts with balances!"
-msgstr "¡No tiene ningúna cuenta de valores con balances!"
+msgstr "¡No tiene ninguna cuenta de valores con dinero!"
```
Comment (ordtrogen) [inline po/es_NI.po]: stock (shares) = acciones?
balance = saldo?

File: po/es_NI.po [hunk 6]
```diff
@@ -923,33 +929,33 @@ msgstr "URL opciones-gnc: mal formados."
...
 msgid "No such entity: %s"
-msgstr ""
+msgstr "No existe la entidad"
 
 #. =================================================================
 #: ../gnucash/gnome/business-urls.c:170
 #, c-format
 msgid "No such owner entity: %s"
-msgstr ""
+msgstr "No existe el dueño de la entidad"
```
Comment (ordtrogen) [inline po/es_NI.po]: "dueño de la entidad" would be "entity owner", but that's different from "owner entity", right?

How about "entidad de dueño"?

**Comment by jralls** [comment]
@ordtrogen please confine your comments on a PR to the changes in that PR. @gsegovia2018 didn't translate any of the other strings and can't answer your questions.

But a general question about the Spanish  translations: There are two, `es.po` by Javier Serrador who has been maintaining it for many years, and `es_NI.po` created by the [Translation Project Spanish Team](https://translationproject.org/team/es.html) in 2001 and apparently never updated since; the translation project no longer tracks it so we can assume that it's abandoned.. At this point it has only 824 not-fuzzy msgstrs while `es.po` is fully up to date and fully translates all strings.

@ordtrogen and @gsegovia2018 are either of you native speakers of Nicaraguan Spanish? If so, does it differ sufficiently from Spanish Spanish to make it worthwhile to maintain a separate translation?

**Comment by jralls** [comment]
@gsegovia2018 you managed to include `.DS_Store`, a MacOS Finder cache file, in your commit. Please amend the commit to remove it and force-push.

**Comment by ordtrogen** [comment]
No, not a native Spanish speaker of any kind. But I do speak enough to ask a few questions. Sorry if I got a little over excited :-)

File: po/es_NI.po [hunk 7]
```diff
@@ -1009,12 +1015,16 @@ msgid ""
 "This transaction needs to be assigned to a Customer. Please choose the "
 "Customer below."
 msgstr ""
+"Esta transacción debe asignarse a un Cliente. Elija el"
+"Cliente a continuación".
```
Comment (jralls) [inline po/es_NI.po]: CI Error: Should be `"Cliente a continuación."`

File: po/es_NI.po [hunk 8]
```diff
@@ -1009,12 +1015,16 @@ msgid ""
...
 
 #: ../gnucash/gnome/dialog-choose-owner.c:85
 msgid ""
 "This transaction needs to be assigned to a Vendor. Please choose the Vendor "
 "below."
 msgstr ""
+"Esta transacción debe asignarse a un Vendedor. Elija el"
+"Vendedor a continuación".
```
Comment (jralls) [inline po/es_NI.po]: CI Error: Should be `"Vendedor a continuación."`

**Comment by jralls** [comment]
@gsegovia2018 Have you abandoned this PR?

**Comment by jralls** [comment]
Closing as abandoned.

**Pull request closed without merging**